src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -26,29 +23,23 @@
   private RelativeEncoder KShooterEncoder;
   private RelativeEncoder KHoodEncoder;
   private MotorController cargo_motorController;
-  private SparkMaxPIDController kCargoController;
-  private RelativeEncoder kCargoEncoder;
-  private NetworkTableEntry sbShooterRPM;
   private double currentRPM;
 
   public ShooterSubsystem() {
-    sbShooterRPM = RobotContainer.debugTab.add("shooterRPM", 0).getEntry();
-
+    SmartDashboard.putNumber("RPMChange", 3000.0);
     aimMode = 4;
     cargo_motorController = new MotorController("Shooter Cargo", Constants.shooterCargoID);
-    kCargoController = cargo_motorController.getPID();
-    kCargoEncoder = cargo_motorController.getEncoder();
-
     shooter_motorController = new MotorController("Shooter", Constants.shooterID, 40, true);
     KShooterController = shooter_motorController.getPID();
     KShooterEncoder = shooter_motorController.getEncoder();
-    KShooterController.setP(6e-4);
+    KShooterController.setP(6e-5);
     KShooterController.setI(1e-6);
     KShooterController.setD(0.0);
-
+    KShooterController.setOutputRange(0, 1);
+    /*
```

Comment:
Let's add TODO comments where we are marking these sections as removed for testing.  
Reason being is that they will then show up in vscode as something to change.